### ISSUE TITLE
lmdb_path

### ISSUE THREAD

**Issue by LYM1111** [clovaai/lffont#23]
您好！请教您一些问题：
1.lmdb_path文件下保存的.mdb文件是包括训练集和测试集的所有数据吗？
2.readme里面叙述的train meta下的"avail"数据应该是什么？我没有理解"avail"，您可以解释一下吗。
3.您是如何拆分部件的，依据的标准是什么？
期待您的回复，谢谢！

**Comment by 8uos** [comment]
Hi.
Thanks for your interest, but I cannot read Chinese.
Can you write your questions in English?

**Comment by LYM1111** [comment]
Hi！Thank you for your reply.
1.Does the dataset.json(json-format meta file ) file include a training set and a test set?What is the number of characters for each font in this file, is it 6654? Can I use 2500?
2.What is the "avail" data under the train meta described in the readme? I don't understand "avail", can you explain it?
3.What is your method of decomposing components?

**Comment by LYM1111** [comment]
What characters are included in the "charlist" in meta/dataset.json？

**Comment by 8uos** [comment]
1. It is okay to use 2500 characters, but all the decomposition information of the characters should be prepared. If all the characters are in the attached decomposition file or you can build your own decomposition file, the number of characters per font does not matter. In our case, each font has a varying number of characters because of the rendering issue and the average is 6,654.
2. "avail" means "available". Sorry for the confusion. The "avail" is a dictionary, whose key is a font and the value is all the available characters of the key font in lmdb.
3. We referred to [here](https://commons.wikimedia.org/wiki/Commons:Chinese_characters_decomposition) to build our decomposition file.
4.  "charlist" is a list of characters that you want to render and dump into the lmdb dataset. If you set the metafile to `{"noto": {"path": "./noto.ttf", "charlist": ["春", "夏", "秋", "冬"]}}`, 4 image("春", "夏", "秋", "冬") in noto font will be rendered and dumped.

**Comment by LYM1111** [comment]
Thank you very much!

**Comment by SanghyukChun** [comment]
Closing the issue, assuming the answer resolves the problem.
Please re-open the issue as necessary.
